### PR TITLE
task 22

### PR DESCRIPTION
**Pull request by losclipp**

### CODE REVIEW

File: 1-select-tasks/SQLQuery22.sql [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+		JOIN [dbo].[Orders] [o] ON [c].[CustomerID] = [o].[CustomerID]
+		JOIN [dbo].[Order Details] [od] ON [o].[OrderID]  = [od].[OrderID]
+		GROUP BY [c].[CustomerID] 
+		) [t1]
+	JOIN ( 
+		SELECT [c].[CustomerID], [od].[ProductID], MAX([od].[UnitPrice]) AS [UnitPrice] 
+		FROM [dbo].[Customers] [c]
+		JOIN [dbo].[Orders] [o] ON [c].[CustomerID] = [o].[CustomerID]
+		JOIN [dbo].[Order Details] [od] ON [o].[OrderID]  = [od].[OrderID]
+		GROUP BY [c].[CustomerID], [od].[ProductID]
```
Comment (AlexMokin) [inline 1-select-tasks/SQLQuery22.sql]: You dont' need group by here

Comment (AlexMokin) [reply 1-select-tasks/SQLQuery22.sql]: neither do you need the subquery

File: 1-select-tasks/SQLQuery22.sql [hunk 2]
```diff
@@ -0,0 +1,32 @@
+SELECT [c].[CompanyName], [p].[ProductName], [od].[UnitPrice] AS [PricePerItem]
```
Comment (AlexMokin) [inline 1-select-tasks/SQLQuery22.sql]: There are duplicates in the result

**Review by Alexander-Shein — COMMENTED** [review]
Good job

**Pull request closed without merging**